data/data/powervs/variables.tf

```diff
@@ -54,38 +54,51 @@ variable "network_name" {
 }
 
 variable "bootstrap" {
-    type    = object({memory = string, processors = string })
-    default = {
-        memory      = "32"
-        processors  = "0.5"
-    }
+  type = object({ memory = string, processors = string })
+  default = {
+    memory     = "32"
+    processors = "0.5"
+  }
 }
 
 variable "proc_type" {
-    type        = string
-    description = "The type of processor mode for all nodes (shared/dedicated)"
-    default     = "shared"
+  type        = string
+  description = "The type of processor mode for all nodes (shared/dedicated)"
+  default     = "shared"
 }
 
 variable "sys_type" {
-    type        = string
-    description = "The type of system (s922/e980)"
-    default     = "s922"
+  type        = string
+  description = "The type of system (s922/e980)"
+  default     = "s922"
 }
 
 # Must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character
 # Length cannot exceed 14 characters when combined with cluster_id_prefix
 variable "cluster_id" {
-    type    = string
-    default = ""
-
-    validation {
-        condition     = can(regex("^$|^[a-z0-9]+[a-zA-Z0-9_\\-.]*[a-z0-9]+$", var.cluster_id))
-        error_message = "The cluster_id value must be a lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character."
-    }
-
-    validation {
-        condition     = length(var.cluster_id) <= 14
-        error_message = "The cluster_id value shouldn't be greater than 14 characters."
-    }
+  type    = string
+  default = ""
+
+  validation {
+    condition     = can(regex("^$|^[a-z0-9]+[a-zA-Z0-9_\\-.]*[a-z0-9]+$", var.cluster_id))
+    error_message = "The cluster_id value must be a lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character."
+  }
+
+  validation {
+    condition     = length(var.cluster_id) <= 14
+    error_message = "The cluster_id value shouldn't be greater than 14 characters."
+  }
+}
+
+variable "powervs_vpc_name" {
+  type        = string
+  description = "Name of the IBM Cloud Virtual Private Cloud (VPC) to setup the load balancer."
+  default     = "powervs-vpc"
```

Comment:
I feel this default is not needed as this is a mandatory param

Reply:
What about the case when we want to create the VPC and not use existing VPC?

Reply:
Then the value should be autogenerated from the cluster-id instead otherwise we won't able to create multiple vpcs with same name in the account.

Reply:
I'll remove the default for now.